src/strategies/index.ts

```diff
@@ -108,6 +108,7 @@ import { strategy as aaveGovernancePower } from './aave-governance-power';
 import { strategy as cake } from './cake';
 import { strategy as planetFinance } from './planet-finance';
 import { strategy as impossibleFinance } from './impossible-finance';
+import { strategy as naga } from './naga';
```

Comment:
Hi.. Please add `naga` to the exported object below, Thanks

Reply:
Thanks!!